apis/vshn/v1/common_types.go

```diff
@@ -261,3 +263,41 @@ func (a *TimeOfDay) AddTime(d time.Duration) TimeOfDay {
 	a.SetTime(a.GetTime().Add(d))
 	return *a
 }
+
+type PodTemplateLabelsManager interface {
+	SetPodTemplateLabels(map[string]string)
+	GetPodTemplateLabels() map[string]string
+	client.Object
+	GetObject() client.Object
+}
```

Comment:
there's interfaces.go file, please put that interafce in there, preferably into InfoGetter as it affects all of our services

Reply:
Unfortunately I cannot use _intefraces.go_ file as it will generate an `import cycle not allowed` error. I am using the definition of the interface directly in the API therefore I cannot import from the comp-function package.